contrib/ci/dependencies.xml

```diff
@@ -766,15 +766,15 @@
     </distro>
     <distro id="debian">
       <control>
-        <version>(>= 0.3.3)</version>
```

Comment:
For now, don't bump this up on Debian & Ubuntu for your PR - it's not in their archives yet (something not yet handled according to current Debian package tracker - https://tracker.debian.org/pkg/libgusb).

It should grab it in subproject form at least to run CI, and it will just remain a blocker to actually go into Debian and Ubuntu later.

Reply:
Thanks. I've done it. Let's see how the builders do.